### ISSUE TITLE
x11-base/xorg-server-1.18.3_p1_p0202:0/1.18.3::unity-gentoo build error

### ISSUE THREAD

**Issue by grray** [shiznix/unity-gentoo#140]
```
/tmp/portage/x11-base/xorg-server-1.18.3_p1_p0202/work/xorg-server-1.18.3/hw/xmir/xmir-input.c: In function ‘xmir_handle_keymap_event’:
/tmp/portage/x11-base/xorg-server-1.18.3_p1_p0202/work/xorg-server-1.18.3/hw/xmir/xmir-input.c:477:5: error: implicit declaration of function ‘mir_keymap_event_get_keymap_buffer’ [-Werror=implicit-function-declaration]
     mir_keymap_event_get_keymap_buffer(ev, (char const **)&buffer, &length);
     ^
```
mir useflag is disabled
```
# emerge -pv xorg-server

These are the packages that would be merged, in order:

Calculating dependencies... done!
[ebuild  r  U ~] x11-base/xorg-server-1.18.3_p1_p0202:0/1.18.3::unity-gentoo [1.17.2_p1_p0901-r1:0/1.17.2::unity-gentoo] USE="dmx glamor ipv6 kdrive suid systemd udev xnest xorg xvfb -doc -libressl% -minimal -mir (-selinux) -static-libs -tslib -unwind -wayland -xephyr (-nptl%*)" 0 KiB
[ebuild  r  U ~] x11-drivers/xf86-video-ati-7.7.0_p1::unity-gentoo [7.5.0_p0_p01::unity-gentoo] USE="glamor udev -mir" 0 KiB
[ebuild  rR    ] x11-drivers/xf86-input-wacom-0.31.0::gentoo  USE="-debug" 0 KiB
[ebuild  r  U ~] x11-drivers/xf86-input-evdev-2.10.3::gentoo [2.9.2::gentoo] 0 KiB

Total: 4 packages (3 upgrades, 1 reinstall), Size of downloads: 0 KiB
```
Maybe I'm using wrong gcc version? I have gcc 5.3.0

**Comment by grray** [comment]
Aha! I solved problem by manually updating mir from 0.17 to 0.21.0_p0_p01_p20160330

**Comment by shiznix** [comment]
Should be fixed in commits 87866f250af3b54313e8971a0d38eb00bc49cda1 and 50ae5783affe2a386cf96510d1b4f8fddcb64f45

Thanks :)
